paasta_tools/utils.py

```diff
@@ -804,7 +804,7 @@ def get_default_cluster_for_service(service):
     return cluster
 
 
-def list_clusters(service=None, soa_dir=DEFAULT_SOA_DIR):
+def list_clusters(service=None, soa_dir=DEFAULT_SOA_DIR, instance_type=None):
```

Comment:
I'd prefer it if you left this function as it is, and either did the filtering wherever you call it, or if you think that the filtering will be done in more than one place, create a wrapper function to do that filtering (like list_chronos_clusters(...) and list_marathon_clusters(...))